analyzer/src/main/kotlin/PackageCurationProvider.kt

```diff
@@ -30,6 +31,11 @@ import org.ossreviewtoolkit.utils.common.NamedPlugin
 interface PackageCurationProviderFactory<CONFIG> : ConfigurablePluginFactory<PackageCurationProvider> {
     companion object {
         val ALL = NamedPlugin.getAll<PackageCurationProviderFactory<*>>()
+
+        fun create(configurations: List<PackageCurationProviderConfiguration>) =
+            // Reverse the list so that curations from providers with higher priority are applied later and can
+            // overwrite curations from providers with lower priority.
+            configurations.filter { it.enabled }.map { ALL.getValue(it.name).create(it.config) }.reversed()
```

Comment:
Use `.asReversed()` to avoid creating a copy.

Reply:
I'm actually unsure about this. Personally, I would expect curation providers to be queried / curations be applied in the order I define them in the configuration, and *not* for the order to express a priority.

Reply:
Do we have other places in the config where order expresses priority that we could align with?

Reply:
I don't think so.

Reply:
> Use `.asReversed()` to avoid creating a copy.

I believe this is the only open remark, otherwise I'd simply approve now.

Reply:
Fixed.